<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Task Automator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="TaskList" name="treeWidget_taskList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>268</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="rootIsDecorated">
     <bool>false</bool>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="headerDefaultSectionSize">
     <number>130</number>
    </attribute>
    <column>
     <property name="text">
      <string>Next Occurrence</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Task</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Adds a new task</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edits the selected task</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Removes the selected tasks</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Selects all tasks in the list</string>
   </property>
  </action>
  <action name="actionSelectNone">
   <property name="text">
    <string>Select None</string>
   </property>
   <property name="toolTip">
    <string>Clears the selection</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
   <property name="toolTip">
    <string>Shows the configuration dialog</string>
   </property>
  </action>
  <action name="actionUserManual">
   <property name="text">
    <string>User Manual</string>
   </property>
   <property name="toolTip">
    <string>Shows the user manual</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Shows authorship information</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>Restore</string>
   </property>
   <property name="toolTip">
    <string>Show Task Automator</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Close Task Automator</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TaskList</class>
   <extends>QTreeWidget</extends>
   <header>tasklist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
